apps/settings/js/panels/achievements/achievements_list.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * UI for Achievements panel's functionality.
+ */
+define(function(require) {
+  'use strict';
+
+  var ListView = require('modules/mvvm/list_view');
+  var ObservableArray = require('modules/mvvm/observable_array');
+  var AchievementItemTemplateFactory = require(
+    'panels/achievements/achievements_template');
+
+  function AchievementsList(panel) {
+    this.listElement = panel.querySelector('.achievements-list');
+    this._listView = ListView(this.listElement, this.achievements,
+      AchievementItemTemplateFactory());
+  }
+
+  AchievementsList.prototype = {
+    get achievements() {
+      this._achievements = this._achievements || ObservableArray([]);
```

Comment:
The getter returns an `ObservableArray` but the setter takes an ordinary array. I would suggest to make `this._achievements` store an ordinary array and use a separate variable for the observable one.